src/main/java/org/mockito/internal/stubbing/BaseStubbing.java

```diff
@@ -15,8 +15,7 @@
 
 public abstract class BaseStubbing<T> implements OngoingStubbing<T> {
 
-    @Override
-    public OngoingStubbing<T> thenReturn(T value) {
+    private OngoingStubbing<T> thenReturn(T value) {
```

Comment:
We are only calling this on line 24, so let's just inline this and remove the method as a whole.

Reply:
Yes, will do that.